<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QWidget" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <widget class="QLineEdit" name="ln_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Enter Url Here</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Save location</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>21</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Download!</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pg_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
